BackofficeBundle/Resources/public/template/Site/sitesView._tpl.html

```diff
@@ -1,5 +1,5 @@
 <div class="top-action-buttons">
-    <% if (!inPlatformContext) { %>
+    <% if (inPlatformContext) { %>
```

Comment:
with this PR :https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2177